src/SourceBuild/Arcade/tools/TextOnlyPackages.csproj

```diff
@@ -0,0 +1,43 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I think it would be valuable to add some documentation targeting devs who seek to add packages.  

1. Guidance on when packages should be added here versus SBRP.  
2. Call out src/SourceBuild/Arcade/tools/SourceBuildArcadeTarball.targets also needs a corresponding entry added.

Reply:
Added guidance at top of the file that this is for text-only packages consumed by installer repo only.